### PR TITLE
IDENTITY-6036: Add "SignJWTWithSPKey" property to identity.xml

### PR DESCRIPTION
**Pull request by piraveena**

### CODE REVIEW

**Comment by piraveena** [comment]
@mefarazath @darshanasbg

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 1]
```diff
@@ -84,6 +84,8 @@
         <OpenIDRememberMeExpiry>7200</OpenIDRememberMeExpiry>
         <!-- To enable or disable openid dumb mode -->
         <DisableOpenIDDumbMode>false</DisableOpenIDDumbMode>
+        <!-- Sign the ID Token with Service Provider Tenant Private Key-->
+        <SignJWTWithSPKey>false</SignJWTWithSPKey>
```
Comment (mefarazath) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml]: This property should come under 


```
<OAuth>
    <OpenIDConnect>
        <SignJWTWithSPKey>false</SignJWTWithSPKey>
```

**Comment by piraveena** [comment]
Fixed requested changes

File: features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml [hunk 2]
```diff
@@ -77,14 +77,14 @@
...
         <OpenIDLoginUrl>${carbon.protocol}://${carbon.host}:${carbon.management.port}/authenticationendpoint/openid_login.do</OpenIDLoginUrl>
-
         <!-- If the users must be prompted for approval -->
         <OpenIDSkipUserConsent>false</OpenIDSkipUserConsent>
         <!-- Expiry time of the OpenID RememberMe token in minutes -->
         <OpenIDRememberMeExpiry>7200</OpenIDRememberMeExpiry>
         <!-- To enable or disable openid dumb mode -->
         <DisableOpenIDDumbMode>false</DisableOpenIDDumbMode>
 
+
```
Comment (omindu) [inline features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml]: Remove the unnecessary new line.

**Comment by piraveena** [comment]
Fixed the requested changes

**Pull request merged**